build/publish.proj

```diff
@@ -1,23 +1,28 @@
 <Project>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.DotNet.BuildTools" Version="$(BuildToolsPackageVersion)" />
```

Comment:
>BuildTools [](start = 48, length = 10)

according to arcade document: [https://github.com/dotnet/arcade/blob/master/Documentation/CorePackages/Publishing.md](https://github.com/dotnet/arcade/blob/master/Documentation/CorePackages/Publishing.md) looks like arcade doesn't support publish packages to our nuget datafeed: [https://dev.azure.com/dnceng/public/_packaging?_a=feed&feed=MachineLearning%40Prerelease](https://dev.azure.com/dnceng/public/_packaging?_a=feed&feed=MachineLearning%40Prerelease)
also don't find it possible to publish symbol files to SymWeb Symbol Server and Msdl Symbol Server so use build tool nuget package here for publish nuget package and symbol purpose as before.